projects/fsastorefrontlib/src/cms-components/my-account/consent-management/consent-management.component.html

```diff
@@ -0,0 +1,70 @@
+<ng-container *ngIf="consentTemplates$ | async as templateList">
+  <div *ngFor="let consentTemplate of templateList">
+    <p>
+      {{ consentTemplate.description }}
+    </p>
+  </div>
+</ng-container>
+
+<ng-container *ngIf="consents$ | async as consents">
+  <ng-container *ngIf="consentsLoaded$ | async">
```

Comment:
Is this needed since in the next container you are checking for length anyway?

Reply:
Done!